tests/conftest.py

```diff
@@ -95,6 +101,50 @@ def test_something(elasticsearch):
     build_index.delete(ignore=404)
 
 
+@runif_bigquery_testing_enabled
```

Comment:
I can mock out the requests to the service. It's possible for this to be set up [with a real project](https://github.com/mozilla/gcp-ingestion/blob/e0bf43ffb3638e093653d74e019a4ba93e44c4c9/.circleci/config.yml#L180-L188) which would fall under the free tier. This would be preferable, since these tests verify the schema via insertion into a newly created table.

Reply:
Perhaps we can have both? Some simpler tests to check what we can (which can always be run locally, even if you're not connected to the internet or have any credentials), and some more complete tests we run in CI?

Reply:
Yep, I wrote a few tests for checking that the client is being hit (or not) after insertion into the main model store. I think using the actual service for verifying correctness is useful, but it does require a bit of configuration.